### DISCUSSION TITLE
Moving to the new .NET Foundation CLA service

### DISCUSSION THREAD

**Discussion by dnfadmin** [dotnet-foundation/Home#83] [Announcements]
Hello everyone
The .NET Foundation is transitioning to a new CLA service and I'm posting here to let you know it's happening. Some background info:

The new CLA service is based off of https://github.com/microsoft/ContributorLicenseAgreement, except it uses the dotnet-policy-service app

- Current contributors do not need to sign the CLA again
- The CLA agreement itself is not changing
- As part of the install/configuration process, the following work items will be completed

1. Create a .github repo if it doesn't exist already
2. Install the dotnet-policy-service app
3. Create PRs for the platformcontext.xml and cla.yml files
4. Disable the existing CLA webhook
5. Push the PRs to enable the new CLA service

That's all there is to it. I'll update this issue once the work is complete

Thanks
Chris
.NET Foundation Operations team

**Comment by barnson** [comment]
How do project admins do this work?

**Comment by lostmsu** [comment]
What is the motivation?

**Comment by lostmsu** [comment]
Can you share the script I could run as org admin so that I would not need to give permissions to DNFAdmin?

**Comment by ChrisSfanos** [comment]
Thanks for reaching out.  
1.  Yes, you can do the work - https://github.com/microsoft/ContributorLicenseAgreement#installation
2. The app is here - https://github.com/apps/dotnet-policy-service
3. Motivation:  the old service is going away so we need to move everyone to the new service.  

Let me know if you have any questions and I can loop in our dev team for any support items

**Reply by ChrisSfanos** [reply]
Hi @lostmsu - checking in on this.  Do you have an ETA when you'll be able to move to the new CLA service? - thanks!

**Reply by lostmsu** [reply]
Just followed the steps, faced https://github.com/microsoft/ContributorLicenseAgreement/issues/121

**Reply by ChrisSfanos** [reply]
Adding @DragosDanielBoia to chime in on that

**Reply by DragosDanielBoia** [reply]
the CLA is already installed https://github.com/dotnet-foundation/.github, will allow today this other org after that should work

**Reply by ChrisSfanos** [reply]
Hi @lostmsu - chatted with Dragos offline and he's aware this repo isn't where you need assistance

**Reply by lostmsu** [reply]
@ChrisSfanos not sure I understood your replies.

**Reply by ChrisSfanos** [reply]
Hey @lostmsu - my understanding from you is you hit the issue above trying to configure the CLA for PythonNET, but Dragos comments were in reference to the CLA already being installed on this repo and not PythonNET.  Was trying to make sure everyone was looking at the correct issue.  Did I misunderstand all of that?

**Reply by lostmsu** [reply]
@ChrisSfanos I got the right response under [the issue I filed](https://github.com/microsoft/ContributorLicenseAgreement/issues/121)

**Reply by ChrisSfanos** [reply]
Ah ok - I hadn't seen that - glad it's covered

**Reply by DragosDanielBoia** [reply]
@lostmsu is done, you will need to re-pushed the files and then should work
